src/sentry/static/sentry/app/components/events/meta/metaProxy.tsx

```diff
@@ -12,6 +14,8 @@ function isAnnotated(meta) {
 }
 
 export class MetaProxy {
+  private local: any;
+
   constructor(local) {
```

Comment:
then add here 
```suggestion
  constructor(local: Local) {
```

Reply:
I don't think we can do that, in theory you can put a metaproxy over every value, integers too